### PR TITLE
I started to create the galaxy class

### PR DESCRIPTION
**Pull request by vcristiani**
This is one of the two main classes of the API. This class will calculate the dynamic quantities needed for dynamic decomposition. - issue #39 .

### CODE REVIEW

File: galaxychop/galaxychop.py [hunk 1]
```diff
@@ -15,7 +15,81 @@
...
+# #####################################################
+
+@attr.s(frozen=True)
+class Galaxy:
+    """This class builds a galaxy object from the masses, positions, and 
+    velocities of the particles (stars, dark matter, and gas).
+
+    Parameters
+    ----------
+    x_s, y_s, z_s: `np.ndarray(n,1), np.ndarray(n,1), np.ndarray(n,1)`
```
Comment (nvillagra) [inline galaxychop/galaxychop.py:30]: Wouldn't it be relevant to put the units here?

**Review by nvillagra — APPROVED** [review]
I just made a silly comment. Structure looks fine.

File: galaxychop/galaxychop.py [hunk 2]
```diff
@@ -15,7 +15,81 @@
...
+    y_g = attr.ib()
+    z_g = attr.ib()
+    vx_g = attr.ib()
+    vy_g = attr.ib()
+    vz_g = attr.ib()
+    m_g = attr.ib()
+
+    components_s = attr.ib(default=None)
+    components_g = attr.ib(default=None)
+    metadata = attr.ib()
```
Comment (BrunoSanchez) [inline galaxychop/galaxychop.py:93]: Does this class have an `__init__` method?

Comment (vcristiani) [reply galaxychop/galaxychop.py:93]: I thought that with the decorator I was using there was no need to put it up. If it is necessary to add it, tell me and I will add it.

Comment (BrunoSanchez) [reply galaxychop/galaxychop.py:93]: That's right on spot. 
You don't need it. :100:

**Comment by vcristiani** [comment]
issue #39 closes with #40

**Pull request merged**